src/Routes/RosPage/RosPage.js

```diff
@@ -181,9 +198,11 @@ class RosPage extends React.Component {
                 stateFilterValue: activeStateFilters
             });
         }
+        // Fixme : how to delete os filter here?
```

Comment:
@apuntamb as discussed can you add starter code based on your understanding for modifying the deletion logic? after that we can go through it together and update the code if needed. Let me know what you think.
Also this code gets called whenever user selects `Reset Filters` option or `x` button on the filter chip.